Adapter_Revit_UI/Adapter_Revit_UI.csproj

```diff
@@ -138,6 +138,10 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\BHoM\Build\Common_oM.dll</HintPath>
     </Reference>
+    <Reference Include="Data_Engine, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Reference should be set to `copy local = false`

Reply:
Fixed